marketplace/cmd/info.go

```diff
@@ -0,0 +1,78 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/dustin/go-humanize"
+	"github.com/google/go-github/v56/github"
+	"github.com/gookit/color"
+	"github.com/ignite/apps/marketplace/pkg/xgithub"
+	"github.com/ignite/cli/ignite/pkg/cliui"
+	"github.com/spf13/cobra"
+)
+
+// NewInfo creates a new info command that shows the details of an ignite app.
+func NewInfo() *cobra.Command {
+	return &cobra.Command{
+		Use:   "info [app-name]",
```

Comment:
app-name or github url?

Reply:
maybe we should call `pkg-url` or something like that. `app-name` seems only the name, not the namespace. wdyt?